CONTRACTS.md

```diff
@@ -6,11 +6,11 @@ the interfaces defined in `packages/cw*`.
 ## Creating a new contract
 
 Use [`cosmwasm-template`](https://github.com/CosmWasm/cosmwasm-template) as a
-basis, in particular the `cosmwasm-plus` branch.
+basis, in particular the `cw-plus` branch.
```

Comment:
ah I need to rename that branch as well. can you make an issue on cw-template so it doesn't get forgotten

Reply:
Done https://github.com/CosmWasm/cosmwasm-template/issues/83